python3Packages.marimo: 0.15.2 -> 0.19.5

dmadisetti opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

erooke reviewed (changes requested):
Looks generally good to me. I've been using this patch locally for a day now and have noticed no regressions.

dmadisetti commented on pkgs/development/python-modules/marimo/default.nix:
```diff
@@ -31,15 +35,27 @@
 }:
 
+let
+  msgspec-m = msgspec.overridePythonAttrs (old: {
```
marimo is happy to cede long term support. It has a 0 build tool requirement policy- so the branch is there to create 3.14 wheels. The alternative is patching the pyproject, which seems reasonable and maybe better

Vortriz replied:
this is not required now given https://github.com/marimo-team/marimo/pull/6797, if i am not mistaken

Vortriz replied:
a gentle ping, since it has been a while. also, please bump this to the latest version.

dmadisetti replied:
`msgspec` is now back upstream, so this can be cleaned up. Thanks for the ping!

GaetanLepage committed:
python3Packages.huggingface-hub: 1.3.2 -> 1.3.3

Diff:
https://github.com/huggingface/huggingface_hub/compare/v1.3.2...v1.3.3

Changelog:
https://github.com/huggingface/huggingface_hub/releases/tag/v1.3.3

bjornfor committed:
pcsclite: 2.3.0 -> 2.4.1

Changelog: https://github.com/LudovicRousseau/PCSC/blob/2.4.1/ChangeLog

Remove patch that is now included in upstream release.

bjornfor committed:
nixos/pcscd: add 'pcscd' user and group

In preparation for pcsclite (>=2.4.0) upgrade, where this user and group
is required.

bjornfor committed:
nixosTests.pcsclite: init

Add a small test to verify that the pcscd socket unit works, and that a
client can connect (which requires auto-starting pcscd.service).

bjornfor committed:
ccid: 1.6.2 -> 1.7.0

Changelog: https://github.com/LudovicRousseau/CCID/releases/tag/1.7.0

Notably, this updates the udev rules to give the 'pcscd' group access to
CCID devices. That's needed for pcsclite>=2.4.0.

Upstream have added installation of the udev rules, but it does so
outside of PREFIX. Keep using our own installation code instead, which
is simpler than patching upstream build system.

bjornfor committed:
pcsclite: add nixosTests.pcsclite to passthru.tests

bjornfor committed:
pcsclite: remove outdated comment

Fixes: 96d6f0a2ddb07916cd1a3230fbe6a21e4dc6b29e ("pcsclite: 2.1.0 -> 2.2.3")

Commit:
python313Packages.google-cloud-iam: 2.20.0 -> 2.21.0

Commit:
python313Packages.google-cloud-kms: 3.7.0 -> 3.9.0

reckenrode committed:
tinyxxd: fix build on Darwin

mweinelt committed:
python3Packages.wheel-filename: 1.4.2 -> 2.1.0

https://github.com/wheelodex/wheel-filename/releases/tag/v2.1.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.wfuzz: 3.1.0 -> 3.1.1

https://github.com/xmendez/wfuzz/releases/tag/v3.1.1

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.widgetsnbextension: 4.0.14 -> 4.0.15

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.wheel-inspect: 1.7.2 -> 1.8.0

https://github.com/wheelodex/wheel-inspect/releases/tag/v1.8.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.wrf-python: 1.4.0 -> 1.4.2

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.wsproto: 1.2.0 -> 1.3.2

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.wtf-peewee: 3.0.6 -> 3.1.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xgrammar: 0.1.24 -> 0.1.31

https://github.com/mlc-ai/xgrammar/releases/tag/v0.1.31

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xcffib: 1.9.0 -> 1.12.0

https://github.com/tych0/xcffib/releases/tag/v1.12.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xattr: 1.2.0 -> 1.3.0

https://github.com/xattr/xattr/blob/v1.3.0/CHANGES.txt

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xkcdpass: 1.20.0 -> 1.30.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xlsxwriter: 3.2.5 -> 3.2.9

https://xlsxwriter.readthedocs.io/changes.html

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xmlsec: 1.3.16 -> 1.3.17

https://github.com/xmlsec/python-xmlsec/releases/tag/1.3.17

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xstatic-bootstrap: 4.5.3.1 -> 5.3.8.0

This commit was automatically generated using update-python-libraries.

Co-authored-by: Michael Daniels <mdaniels5757@gmail.com>

mweinelt committed:
python3Packages.xtensor-python: 0.28.0 -> 0.29.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xxhash: 3.5.0 -> 3.6.0

https://github.com/ifduyue/python-xxhash/blob/v3.6.0/CHANGELOG.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xvfbwrapper: 0.2.16 -> 0.2.18

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.yangson: 1.6.3 -> 1.6.6

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.xyzservices: 2025.4.0 -> 2025.11.0

https://github.com/geopandas/xyzservices/releases/tag/2025.11.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.yaspin: 3.1.0 -> 3.4.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.ydata-profiling: 4.18.0 -> 4.18.1

https://github.com/ydataai/ydata-profiling/releases/tag/v4.18.1

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.youtube-search: 2.1.2 -> 2.2.0

This commit was automatically generated using update-python-libraries.

Co-authored-by: Michael Daniels <mdaniels5757@gmail.com>

mweinelt committed:
python3Packages.yfinance: 0.2.66 -> 1.0

https://github.com/ranaroussi/yfinance/blob/1.0/CHANGELOG.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zc-buildout: 4.1.12 -> 5.1.1

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zc-lockfile: 3.0.post1 -> 4.0

https://github.com/zopefoundation/zc.lockfile/blob/4.0/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zconfig: 4.2 -> 4.3

https://github.com/zopefoundation/ZConfig/blob/4.3/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zeep: 4.3.1 -> 4.3.2

https://github.com/mvantellingen/python-zeep/releases/tag/4.3.2

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zeroc-ice: 3.7.10.1 -> 3.8.0.post1

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zodbpickle: 4.2 -> 4.3

https://github.com/zopefoundation/zodbpickle/blob/4.3/CHANGES.rst

This commit was automatically generated using update-python-libraries.

Co-authored-by: Michael Daniels <mdaniels5757@gmail.com>

mweinelt committed:
python3Packages.zimports: 0.6.2 -> 0.7.0

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-event: 6.0 -> 6.1

https://github.com/zopefoundation/zope.event/blob/6.1/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-i18nmessageid: 8.0 -> 8.2

https://github.com/zopefoundation/zope.i18nmessageid/blob/8.2/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-hookable: 8.0 -> 8.2

https://github.com/zopefoundation/zope.hookable/blob/8.2/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-proxy: 7.0 -> 7.1

https://github.com/zopefoundation/zope.proxy/blob/7.1/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-security: 8.1 -> 8.3

https://github.com/zopefoundation/zope.security/blob/8.3/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-interface: 8.0.1 -> 8.2

https://github.com/zopefoundation/zope.interface/blob/8.2/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zope-testing: 6.0 -> 6.1

https://github.com/zopefoundation/zope.testing/blob/6.1/CHANGES.rst

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.zstd: 1.5.7.2 -> 1.5.7.3

This commit was automatically generated using update-python-libraries.

mweinelt committed:
python3Packages.traitlets: disable failing tests

mweinelt committed:
python3Packages.cffi: disable failing tests

mweinelt committed:
python3Packages.sqlalchemy: 2.0.45 -> 2.0.46

https://github.com/sqlalchemy/sqlalchemy/releases/tag/rel_2_0_46

mweinelt committed:
python3Packages.scipy: fix build

dotlambda committed:
python3Packages.pandas: fix src reproducibility

mweinelt committed:
python3Packages.pyeiscp: drop

Was previously a home-assistant dependency, but dropped mid 2025.

mweinelt committed:
python3Packages.pytest-regressions: disable failing tests

Various dataframes have representation and compat issues.

mweinelt committed:
python3Packages.testscenarios: disable tests

They stopped being compatible with testtools 2.8, the patch to fix does
not apply and the repo has no useful tags.

mweinelt committed:
python3Packages.requests-mock: fix tests

mweinelt committed:
python3Packages.deepdiff: provide pytz for tests

and bring them in alphabetical order.

mweinelt committed:
globus-cli: 3.38.0 -> 3.41.0

https://github.com/globus/globus-cli/releases/tag/3.41.0

mweinelt committed:
python3Packages.pyee: fix tests

dmadisetti commented:
Release expected later today- so I'll update for 0.19.5. However, 3.14 will be broken until msgspec is updated

mweinelt committed:
python3Packages.readme-renderer: fix docutils compat

mweinelt committed:
python3Packages.ml-dtypes: relax setuptools constraint

mweinelt committed:
python3Packages.levenshtein: fix cython 3.2 compat

dotlambda committed:
python3Packages.google-genai: unpin websockets

dotlambda committed:
python3Packages.bumble: unpin tomli

mweinelt committed:
python3Packages.pillow-heif: 1.1.1 -> 1.2.0

https://github.com/bigcat88/pillow_heif/releases/tag/v1.2.0

erooke reviewed (commented):
> However, 3.14 will be broken until msgspec is updated

As it is right now it won't build locally for me on 3.14, 3.13, 3.12, or 3.11. I'm assuming this is more or less expected:

```
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for marimo-0.19.5-py3-none-any.whl
  - msgspec>=0.20.0 not satisfied by version 0.19.0
```

The changes look good to me but I haven't run it locally due to this.

edit: it's showing that I requested changes, which was not my intention :sweat_smile:

mweinelt committed:
Revert "python3Packages.pandas: 2.3.3 -> 3.0.0"

This reverts commits 2ad5e32bd7e0f48571f8c7d198be1bcc1e6cee20 and
178b97cdc815288976dcc16af8eb7ad319a07f9c.

Pandas was just released as this cycle began and we picked it up less
than two hours after its release. This is too early for various packages
that couldn't yet deal with its breaking changes.

mweinelt committed:
awscli: 1.42.18 -> 1.44.21

https://github.com/aws/aws-cli/blob/1.44.21/CHANGELOG.rst

mweinelt committed:
python3Packages.filterpy: fix numpy 2.4 compat

mweinelt committed:
python3Packages.wrapt: fix pytest 9 compat

Instead of upgrading, because the opentelemetry packages don't support
wrapt 2.0.

mweinelt committed:
python3Packages.cherrypy: disable failing test

dotlambda committed:
python3Packages.sagemaker: 2.254.1 -> 2.256.1

Diff: https://github.com/aws/sagemaker-python-sdk/compare/v2.254.1...v2.256.1

Changelog: https://github.com/aws/sagemaker-python-sdk/blob/v2.256.1/CHANGELOG.md

qweered committed:
python313Packages.pygame: fix build by skipping tests

mweinelt committed:
python3Packages.overrides: fix pytest9 compat

dotlambda committed:
python3Packages.home-assistant-chip-core: reduce dependencies

mweinelt committed:
python3Packages.home-assistant-chip-core: remove ecdsa dependency

In nixpkgs ecdsa is marked vulnerable because the upstream author
declared the library to be a testing and teaching tool and not fit for
production usage.

In project-chip ecdsa is only used for testing purposes, so it is
apparently safe to remove.

dotlambda committed:
python3Packages.securetar: update repo URL

dotlambda committed:
python3Packages.fs: disable tests relying on the removed pyftpdlib.test

mweinelt committed:
python3Packages.readchar: fix pytest 9 compat

mweinelt committed:
python3Packages.uv-build: 0.9.9 -> 0.9.26

https://github.com/astral-sh/uv/blob/0.9.26/CHANGELOG.md

mweinelt committed:
python3Packages.boltons: fix pytest 9 compat

mweinelt committed:
python3Packages.langchain-tests: relax pytest & vcrpy constraint

mweinelt committed:
python3Packages.krb5: relax cython constraint

mweinelt committed:
python3Packages.dateparser: fix ruamel.yaml 0.19 compat

mweinelt committed:
python3Packages.specfile: fix src hash

dotlambda committed:
python3Packages.dill: fix changelog

doronbehar committed:
python3Packages.numba: use finalAttrs pattern

doronbehar committed:
python3Packages.numba: remove unused expression arguments

doronbehar committed:
python3.pkgs.numba: use writableTmpDirAsHomeHook

doronbehar committed:
python3.pkgs.numba: categorise expression arguments

doronbehar committed:
python3.pkgs.numba: support Numpy 2.4

mweinelt committed:
python3Packages.dulwich: 0.24.1 -> 1.0.0

https://github.com/jelmer/dulwich/blob/dulwich-dulwich-1.0.0/NEWS

mweinelt committed:
fixup! python3Packages.types-aiobotocore: 3.1.0 -> 3.1.1

mweinelt committed:
xandikos: 0.2.12 -> 0.3.4

https://github.com/jelmer/xandikos/blob/v0.3.4/NEWS

mweinelt committed:
poetry: 2.2.1 -> 2.3.1

https://github.com/python-poetry/poetry/blob/2.3.1/CHANGELOG.md

mweinelt committed:
playwright: provide expat for chromium-headless-shell

qweered committed:
python313Packages.pysdl2: skip failing test_SDL_GetSetWindowMouseRect

mweinelt committed:
python3Packages.azure-core: pin to pytest 8.3

Tries to set a mark on a fixture, which is not supported.

mweinelt committed:
python3Packages.extension-helpers: disable failing test

mweinelt committed:
python3Packages.eth-abi: relax parsimonious constraint

mweinelt committed:
python3Packages.flet: various fixes to the build

GaetanLepage committed:
python3Packages.transformers: 4.57.6 -> 5.0.0

Diff: https://github.com/huggingface/transformers/compare/v4.57.6...v5.0.0

Changelog: https://github.com/huggingface/transformers/releases/tag/v5.0.0

GaetanLepage committed:
python3Packages.lerobot: 0.4.2 -> 0.4.3

Diff: https://github.com/huggingface/lerobot/compare/v0.4.2...v0.4.3

Changelog: https://github.com/huggingface/lerobot/releases/tag/v0.4.3

GaetanLepage committed:
python3Packages.torchtune: remove huggingface-hub optional deps

mweinelt committed:
python3Packages.tensorflow: drop dependency on keras and libclang

We now check runtime requirements for binary wheels and these two
libraries apparently were missing all along.

That did not cause any issues and I think we can skip them until someone
complains about it.

dotlambda committed:
python3Packages.asyncpg: use pytest_8_3

Otherwise the tests fail with errors like

    execnet.gateway_base.DumpError: can't serialize <class 'decimal.Decimal'>

dotlambda committed:
python3Packages.wcwidth: 0.2.14 -> 0.4.0

Diff: https://github.com/jquast/wcwidth/compare/0.2.14...0.4.0

Changelog: https://github.com/jquast/wcwidth/releases/tag/0.4.0

mweinelt committed:
python3Packages.seaborn: pin to pytest 8.3, fix numpy 2.4 compat

dotlambda committed:
python3Packages.ftfy: skip test broken by wcwidth 0.2.14

mweinelt committed:
python3Packages.bleach: prune deps, refactor

mweinelt committed:
pdm: 2.26.2 -> 2.26.6

https://github.com/pdm-project/pdm/releases/tag/2.26.6

mweinelt committed:
python3Packages.samplerate: fix numpy 2.4 compat

mweinelt committed:
python3Packages.gradio-client: fix tests

mweinelt committed:
fixup! python3Packages.array-api-compat: 1.12 -> 1.13

mweinelt committed:
python3Packages.asdf-astropy: fix tests

GaetanLepage committed:
python3Packages.huggingface-hub: 1.3.3 -> 1.3.4

Diff: https://github.com/huggingface/huggingface_hub/compare/v1.3.3...v1.3.4

Changelog: https://github.com/huggingface/huggingface_hub/releases/tag/v1.3.4

mweinelt committed:
python3Packages.aocd-example-parser: 2024.12.25 -> 2025.12.12

https://github.com/wimglenn/aocd-example-parser/compare/c86bfc714d2f413965a46a2caf3483e823ea9ade...c86bfc714d2f413965a46a2caf3483e823ea9ade

dotlambda committed:
python3Packages.asyncclick: skip failing tests

mweinelt committed:
python3Packages.archinfo: 9.2.193 -> 9.2.154

https://github.com/angr/archinfo/compare/v9.2.193...v9.2.154

mweinelt committed:
python3Packages.py-evm: ignore deprecation warnings

mweinelt committed:
python3Packages.skia-pathops: 0.8.0.post2 -> 0.9.1

mweinelt committed:
python3Packages.xarray: disable failing tests

sternenseemann committed:
haskell.compiler.*: fix user's guide build with Sphinx 9

mweinelt committed:
python3Packages.testresources: disable failing tests

mweinelt committed:
python3Packages.cython_3_1: init at 3.1.8

mweinelt committed:
python3Packages.pandas: build with cython 3.1

With Cython 3.2 the serialization of some objects fails and breaks tests
downstream in dask.

dotlambda committed:
python3Packages.ocrmypdf: 16.13.0 -> 17.0.1

Diff: https://github.com/ocrmypdf/OCRmyPDF/compare/v16.13.0...v17.0.1

Changelog: https://github.com/ocrmypdf/OCRmyPDF/blob/v17.0.1/docs/release_notes.md

SFrijters committed:
ffado: move NIX_LDFLAGS into env for structuredAttrs

mdaniels5757 committed:
python3Packages.bleach: relax tinycss2 dep

mweinelt committed:
python3Packages.pytest-sugar: fix pytest 9 compat

mweinelt committed:
rdkafka: 2.11.1 -> 2.13.0

https://github.com/confluentinc/librdkafka/compare/v2.11.1...v2.13.0

mweinelt committed:
python3Packages.python-creole: disable failing test

mweinelt committed:
python3Packages.pyppeteer: relax deps

mweinelt committed:
python3Packages.cadwyn: 5.6.1 -> 6.0.1

https://github.com/zmievsa/cadwyn/releases/tag/6.0.1

mweinelt committed:
python3Packages.celery: disable failing tests

This is likely an API change in pymongo between 4.13.2 and 4.16.0.

mweinelt committed:
python3Packages.rq: fix redis over tcp/ip tests

mweinelt committed:
music-assistant: relax aiofiles constraint

mweinelt committed:
pretix: fix overriden stripe package build

mweinelt committed:
python3Packages.pytest-aiohttp: ignore deprecation warnings

This one is for deprecations in the async module, raised in 3.14 for
removal in 3.16.

mdaniels5757 committed:
python3Packages.scspell: 2.3 -> 2.3.0-unstable-2025-04-06

Fixes build with python314.

mweinelt committed:
python3Packages.fastapi-github-oidc: disable failing test

mweinelt committed:
python3Packages.langchain-classic: pin to pytest 8.3

Sets marks on fixtures, which errors out on pytest 9.

mweinelt committed:
python3Packages.shap: relax numba and llvmlite constraints

mweinelt committed:
python3Packages.dask-image: fix tests

mweinelt committed:
python3Packages.apache-beam: relax httplib2 constraint

dotlambda committed:
python3Packages.bibtexparser: 1.4.3 -> 1.4.4

Diff: https://github.com/sciunto-org/python-bibtexparser/compare/v1.4.3...v1.4.4

dotlambda committed:
python3Packages.biopython: mark broken

mdaniels5757 committed:
python3Packages.tbm-utils: fix tests

dotlambda committed:
python3Packages.fava: 1.30.7 -> 1.30.11

Diff: https://github.com/beancount/fava/compare/v1.30.7...v1.30.11

dotlambda committed:
python3Packages.fiona: replace deprecated parseString with parse_string

bjornfor committed:
nixos/pcscd: add ccid udev rules

In preparation for pcsclite (>= 2.4.0) upgrade, where the ccid
(>= 1.7.0) udev rules are needed to provide smart card reader access to
the 'pcscd' group.

mweinelt committed:
python3Packages.pyspcwebgw: pin to pytest 8.3

Sets a mark on a fixture, which is unsupported in pytest 9.

mweinelt committed:
python3Packages.pysignalr: relax websockets constraint

mweinelt committed:
python3Packages.nice-go: relax aiobotocore constraint

mweinelt committed:
python3Packages.gspread: pin to pytest 8.3

Tries to apply a mark fo a fixture, which is not allowed in pytest 9.

mweinelt committed:
python3Packages.bluecurrent-api: relax websockets constraint

mweinelt committed:
python3Packages.bx-py-utils: disable failing tests

mdaniels5757 committed:
python3Packages.sudachipy: disable with python 3.14+

mweinelt committed:
[python-updates] major updates 2026-01-21 (#482364)

dotlambda committed:
python3Packages.ocrmypdf: 16.13.0 -> 17.0.1 (#485418)

FliegendeWurst committed:
sqlite: enable Percentile extension (#483781)

dotlambda committed:
python3Packages.syrupy: 5.0.0 -> 5.1.0 (#484288)

dotlambda committed:
python3Packages.markdown: 3.10.0 -> 3.10.1 (#485386)

dotlambda committed:
python3Packages.multidict: 6.7.0 -> 6.7.1 (#484097)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge staging-next into staging

Sigmanificient committed:
build-support: set pname and version

philiptaron committed:
tinyxxd: fix build on Darwin (#485976)

bjornfor committed:
ccid: prepare udev rules for use in NixOS

There is a RUN+="/usr/sbin/Kobil_mIDentity_switch" directive in the udev
rules file, and since we don't have that program (we don't build ccid
with "extras"), NixOS fails use it:

  Checking that all programs called by absolute paths in udev rules exist... FAIL
  /usr/sbin/Kobil_mIDentity_switch is called in udev rules but is not executable or does not exist

Fix it by disabling that rule for now.

SFrijters committed:
libavc1394: move NIX_LDFLAGS into env for structuredAttrs

SFrijters committed:
libvdpau: move NIX_LDFLAGS into env for structuredAttrs

bjornfor committed:
pcsclite: 2.3.0 -> 2.4.1 (#477133)

nixpkgs-ci[bot] committed:
Merge master into staging-next

mweinelt committed:
python3Packages.paho-mqtt: disable failing test

Swarsel committed:
oauth2-proxy: 7.14.0 -> 7.14.2

mdaniels5757 committed:
Merge remote-tracking branch 'upstream/staging-next' into staging

Swarsel committed:
oauth2-proxy: add swarsel as maintainer

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

jopejoe1 committed:
build-support: set pname and version (#486003)

doronbehar committed:
python3.pkgs.xarray: fix disabled tests attribute & comments

A fixup to:
https://github.com/NixOS/nixpkgs/commit/0871428740c879a3eef0f399e8265ab6947df500

doronbehar committed:
python3.pkgs.scipy: revive Numpy 1 pyproject.toml substitution

This commit reverts most of 3cadb0cf908455a617a29bc08ab317a28aecec11.
See comment within.

SFrijters committed:
libxmu: 1.3.0 -> 1.3.1

Fixes compilation for 32-bit targets:
https://gitlab.freedesktop.org/xorg/lib/libxmu/-/commit/3d207f8600adbc8fad6f5a5daaa66bf3961d9bdd

jopejoe1 committed:
libxmu: 1.3.0 -> 1.3.1 (#486231)

layus committed:
auto-patchelf: add --structured-logs feature, add test (#483740)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

mweinelt committed:
python3.pkgs.scipy: revive Numpy 1 pyproject.toml substitution (#486203)

mweinelt committed:
python3.pkgs.xarray: fix disabled tests attribute & comments (#486200)

SuperSandro2000 committed:
oauth2-proxy: 7.14.0 -> 7.14.2; add maintainer (#486125)

dotlambda committed:
libadwaita: 1.8.3 -> 1.8.4

Diff: https://gitlab.gnome.org/GNOME/libadwaita/-/compare/1.8.3...1.8.4

Changelog: https://gitlab.gnome.org/GNOME/libadwaita/-/blob/1.8.4/NEWS

fabianhjr committed:
gi-docgen: 2025.5 → 2026.1 (#484320)

SuperSandro2000 committed:
lcms: 2.17 -> 2.18 (#478580)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

SFrijters committed:
python3Packages.fastjsonschema: fix build for structuredAttrs

NickCao committed:
fftw: enable neon for aarch64 (#484649)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge staging-next into staging

SigmaSquadron committed:
stdenv: add pname and version

Signed-off-by: Fernando Rodrigues <alpha@sigmasquadron.net>

nixpkgs-ci[bot] committed:
Merge master into staging-next

trofi committed:
tinyxxd: 1.3.8 -> 1.3.10

Changes: https://github.com/xyproto/tinyxxd/compare/v1.3.8...v1.3.10

K900 committed:
qt6: 6.10.1 -> 6.10.2

K900 committed:
qt6: 6.10.1 -> 6.10.2 (#486498)

ulrikstrid committed:
ocamlPackages.uucp: modernize (#484536)

wolfgangwalther committed:
ffado: move NIX_LDFLAGS into env for structuredAttrs (#486041)

wolfgangwalther committed:
libvdpau: move NIX_LDFLAGS into env for structuredAttrs (#486068)

wolfgangwalther committed:
libavc1394: move NIX_LDFLAGS into env for structuredAttrs (#486093)

wolfgangwalther committed:
python3Packages.fastjsonschema: fix build for structuredAttrs (#486376)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

K900 committed:
glslang: 16.1.0 -> 16.2.0

Diff: https://github.com/KhronosGroup/glslang/compare/16.1.0...16.2.0

K900 committed:
vulkan-headers: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-Headers/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-loader: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-Loader/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-validation-layers: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-ValidationLayers/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-tools: 1.4.335 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-Tools/compare/vulkan-sdk-1.4.335...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-tools-lunarg: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/LunarG/VulkanTools/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-extension-layer: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-ExtensionLayer/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-utility-libraries: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/Vulkan-Utility-Libraries/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
vulkan-volk: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/zeux/volk/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
spirv-headers: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/SPIRV-Headers/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

K900 committed:
spirv-cross: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/SPIRV-Cross/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

Changelog: https://github.com/KhronosGroup/SPIRV-Cross/releases/tag/1.4.341.0

K900 committed:
spirv-tools: 1.4.335.0 -> 1.4.341.0

Diff: https://github.com/KhronosGroup/SPIRV-Tools/compare/vulkan-sdk-1.4.335.0...vulkan-sdk-1.4.341.0

philiptaron committed:
tinyxxd: 1.3.8 -> 1.3.10 (#486485)

r-ryantm committed:
libxmlb: 0.3.24 -> 0.3.25

K900 committed:
Vulkan 1.4.341 (#486573)

philiptaron committed:
stdenv: add pname and version (#486470)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

NickCao committed:
libxmlb: 0.3.24 -> 0.3.25 (#486581)

nixpkgs-ci[bot] committed:
Merge master into staging-next

mdaniels5757 committed:
Merge remote-tracking branch 'upstream/staging-next' into staging

vbgl committed:
ocamlPackages.cmdliner: 1.3.0 → 2.1.0 (#484655)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

jtojnar committed:
libadwaita: 1.8.3 -> 1.8.4 (#486330)

jtojnar committed:
gdk-pixbuf: 2.44.4 → 2.44.5 (#485616)

r-ryantm committed:
libbpf: 1.6.2 -> 1.6.3

vcunat committed:
libbpf: 1.6.2 -> 1.6.3 (#486871)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge staging-next into staging

marcin-serwin committed:
Revert "kdePackages.kdenlive: add workaround for broken shaderc pc file"

This reverts commit e848373e2c049b8df7dd5d350e2d1ea32f850e4a.

leonklingele committed:
go_1_25: 1.25.6 -> 1.25.7

marcin-serwin committed:
python3Packages.mpv: don't use xorg package set

The package set has been deprecated in
https://github.com/NixOS/nixpkgs/pull/479724 and its usage is causing
eval failures on staging.

Signed-off-by: Marcin Serwin <marcin@serwin.dev>

jopejoe1 committed:
python3Packages.mpv: don't use xorg package set (#486969)

K900 committed:
Revert "kdePackages.kdenlive: add workaround for broken shaderc pc file" (#486958)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/staging-next' into staging

katexochen committed:
go_1_25: 1.25.6 -> 1.25.7 (#486964)

dmadisetti committed:
python3Packages.marimo: 0.19.4 -> 0.19.7
